common/src/test/java/com/hyperwallet/android/common/util/DateUtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package com.hyperwallet.android.common.util;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import java.util.Date;
+
+public class DateUtilsTest {
```

Comment:
we need a test to confirm that the gmt date returning from the server is actually being converted to the devices locale/timezone

Reply:
hmm formatting of string to different locale is not supported in unit test, that means it will only work on automation test, probably we can add that to automation test

Reply:
found a way, we might be mis understanding androids formatter